src/main/java/com/google/cloud/genomics/dataflow/pipelines/CountReads.java

```diff
@@ -57,21 +57,8 @@
 import java.util.logging.Logger;
 
 /**
- * Simple read counting pipeline, intended as an example for reading data from 
+ * Simple read counting pipeline, intended as an example for reading data from
  * APIs OR BAM files and invoking GATK tools.
- *
- * Specify either ReadGroupSet or BAMFilePath.
- *
- * Example command line (you have to fill in the env. variables; BAMFilePath is of the "gs://foo/bar" format):
- * java -cp target/google-genomics*jar com.google.cloud.genomics.dataflow.pipelines.CountReads \
- *   --project=$PROJECT_ID \
- *   --stagingLocation=$STAGING \
- *   --secretsFile=$CLIENT_SECRETS \
- *   --references=$DESIRED_CONTIGS \
- *   --BAMFilePath=$BAM_FILE_PATH \
- *   --output=$OUTPUT
- *
- * See src/main/scripts/count_reads.sh for more detail.
  */
 public class CountReads {
```

Comment:
Perhaps in the javadoc above, add a link to the doc at
http://google-genomics.readthedocs.org/en/latest/use_cases/analyze_reads/count_reads.html ?